src/PersonObjects/Player/PlayerObject.ts

```diff
@@ -132,6 +132,9 @@ export class PlayerObject extends Person implements IPlayer {
   sourceFileLvl = generalMethods.sourceFileLvl;
   applyEntropy = augmentationMethods.applyEntropy;
   focusPenalty = generalMethods.focusPenalty;
+  bitNodeFinishedState = generalMethods.bitNodeFinishedState;
+  canAccessBitNodeFeature = generalMethods.canAccessBitNodeFeature;
```

Comment:
What is the purpose of moving these methods onto `Player`? It's a god-object, and IMO we should avoid expanding it further. I can see the logic for sharing `bitNodeFinishedState` (although there are other ways it could be shared, probably), but the other two are only used for the achievements I think?